addons/Paragraph/src/presenter.js

```diff
@@ -246,7 +258,8 @@ function AddonParagraph_create() {
             hasDefaultFontSize = false,
             layoutType = model["Layout Type"] || "Default",
             title = model["Title"],
-            manualGrading = ModelValidationUtils.validateBoolean(model["Manual grading"]);
+            manualGrading = ModelValidationUtils.validateBoolean(model["Manual grading"]),
+            weight = model['Weight'];

```

Comment:
W metodzie upgradeModel trzeba jeszcze dodać do modelu to property. 
W przypadkach gdy wczytane zostaną stare lekcje to pole w modelu będzie miało wartość undefined - w metodzie ModelValidationUtils.isStringEmpty ta sytuacja jest sprawdzana, ale do zwracanego obiektu przypisywana jest wartość undefined.
W przyszłości jeżeli ktoś będzie z wykorzystywał to property, to może się natknąć na wartość której nie oczekuje.
Można się wzorować na commice 7aa406b8f45e6ca18ad1b1b90eae64c6fbac5032.
Analogicznie zrobić to dla drugiego rodzaju paragrafu.

Reply:
Mała pomyłka - chciałem dać "Request changes" :)